fe/functional.py

```diff
@@ -50,3 +58,24 @@ def U(coords, params, box, lam):
         return np.sum(np.array([U_i(coords, p_i, box, lam) for (U_i, p_i) in zip(U_s, params)]))
 
     return U
+
+
+def construct_differentiable_interface_cpp(unbound_potentials, params, precision=np.float32):
```

Comment:
Did we want to rename this to something else? eg. _fast or _sum or _optimized?

Reply:
Thanks, I like all of those better than `_cpp`. If we don't have an existing convention maybe I'll choose `_fast`.

Reply:
Changed in 1592167 (and 1ed2c2b)